src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1284,8 +1281,9 @@ export class EditDataGridPanel extends GridParentComponent {
 			} else {
 				valueToDisplay = value;
 			}
-			valueToDisplay = escape(valueToDisplay.length > 250 ? valueToDisplay.slice(0, 250) + '...' : valueToDisplay);
+			valueToDisplay = newlinePattern ? valueToDisplay.replace(/\u0000/g, newlinePattern) : valueToDisplay;
```

Comment:
Can this be passed into Services.getCellDisplayValue instead? I believe this should be consistent from everywhere a table cell is displayed..